core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -608,6 +608,18 @@ portfolio.pending.step2_seller.waitPayment.msg=The deposit transaction has at le
 portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nIf the trade has not been completed on {1} the arbitrator will investigate.
 portfolio.pending.step2_seller.openForDispute=The BTC buyer has not started his payment!\nThe max. allowed period for the trade has elapsed.\nYou can wait longer and give the trading peer more time or contact the arbitrator for opening a dispute.
 
+tradeChat.chatWindowTitle=Chat window for trade with ID ''{0}''
+tradeChat.openChat=Open chat window
+tradeChat.rules=You can communicate with your trade peer to resolve potential problems with this trade. It is not mandatory to reply in \
+  the chat. If a trader violates the below rules, open a dispute with 'cmd + o' and report it to the arbitrator.\n\n\
+  Chat rules:\n\
+  \t● Do not send links to unknown URLs (risk of malware). Links to a block explorer are permitted.\n\
+  \t● Do not try to motivate peer to trade outside of Bisq (no security).\n\
+  \t● Keep conversation friendly and respectful.\n\
+  \t● Do not engage in any form of social engineering scam attempts.\n\
+  \t● Respect if the peer is not responding and prefers to not communicate via the chat.\n\
```

Comment:
```suggestion
  \t● If a peer is not responding and prefers to not communicate via chat, respect their decision.\n\
```

Reply:
I would also add an additional rule, as it comes up quite a bit, but is a huge problem for privacy:

```
\t● Do not attempt to contact the trading peer outside of this chat.\n\
```

Reply:
Do not copy and paste your seed words, private keys, passwords, or other sensitive information into this chat!

Reply:
> Do not attempt to contact the trading peer outside of this chat

Not so sure if that should be a strict rule as in cases of network instability users do communicate via email if the email is known. I would say lets leave that to common sense of users...

Reply:
Updated version:

```
tradeChat.rules=You can communicate with your trade peer to resolve potential problems with this trade.\n\
  It is not mandatory to reply in the chat.\n\
  If a trader violates the below rules, open a dispute with 'Cmd/Ctrl + o' and report it to the arbitrator.\n\n\
  Chat rules:\n\
  \t● Do not send any links (risk of malware). You can send the transaction ID and the name of a block explorer.\n\
  \t● Do not send your seed words, private keys, passwords or other sensitive information!\n\
  \t● Do not encourage trading outside of Bisq (no security).\n\
  \t● Do not engage in any form of social engineering scam attempts.\n\
  \t● If a peer is not responding and prefers to not communicate via chat, respect their decision.\n\
  \t● Keep conversation scope limited to the trade. This chat is not a messenger replacement or trollbox.\n\
  \t● Keep conversation friendly and respectful.
```